### ISSUE TITLE
Docker Desktop App and Docker Desktop Service Crashing Daily

### ISSUE THREAD

**Issue by adamhwang** [docker/for-win#4916]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 4E5C73D8-4528-47CC-A47F-1FF006AD0585/20191011122717

### Expected behavior
Docker should run without crashing on Windows with 3 linux containers (influxdb, mssql-servicer-linux, and redis)

### Actual behavior
Docker is consistently silently crashing daily. Docker Desktop App is unresponsive when attempting to Quit through the context menu, and the only way to reset Docker is by killing the process and restarting the App and Service.

### Information

  - Windows Version: Windows 10 Pro, 1803, 17134.1069
  - Docker Desktop Version: 2.1.0.2 (37877)
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: No

This appears to be a new issue introduced in the last week or so. I've been able to run these same 3 containers for several weeks prior without issue. Between 10/4/2019 and 10/8/2019 the server had an issue preventing remote access: 

```The security manager (SAM) or local security authority (LSA) server was in the wrong state to perform the security operation```

After physically restarting the server on 10/8/2019, the above error disappeared but the Docker Desktop App and Service became unstable and crashed daily. So it appears to have started between 10/4/2019 and 10/8/2019. The last possibly relevant log message appears to be:

```
[03:31:11.650][GoBackendProcess  ][Info   ] runtime: VirtualAlloc of 0 bytes failed with errno=1455
[03:31:11.650][GoBackendProcess  ][Info   ] fatal error: runtime: failed to commit pages
```

### Steps to reproduce the behavior

Not sure it's possible to reproduce exactly. It crashes consistently daily but generally only after several hours of running. I'm running the following 3 containers:
  1. `docker run -v D:/databases:/var/opt/mssqlhost -e "ACCEPT_EULA=Y" -e "MSSQL_SA_PASSWORD=xxxx" -p 1433:1433 --restart unless-stopped --expose 1433 --name mssql -d microsoft/mssql-server-linux:latest`
  2. `docker run -d --name redis -p 6379:6379 -v --restart unless-stopped D:/docker/redis.conf:/redis.conf redis redis-server /redis.conf --appendonly yes`
  3. `docker run -d --name influxdb-local -p 8086:8086 --restart unless-stopped influxdb`
